Examples/MAX78002/CameraIF_Debayer/utils.c

```diff
@@ -0,0 +1,177 @@
+/*******************************************************************************
+* Copyright (C) Maxim Integrated Products, Inc., All Rights Reserved.
+*
+* Permission is hereby granted, free of charge, to any person obtaining a
+* copy of this software and associated documentation files (the "Software"),
+* to deal in the Software without restriction, including without limitation
+* the rights to use, copy, modify, merge, publish, distribute, sublicense,
+* and/or sell copies of the Software, and to permit persons to whom the
+* Software is furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included
+* in all copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+* OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+* IN NO EVENT SHALL MAXIM INTEGRATED BE LIABLE FOR ANY CLAIM, DAMAGES
+* OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+* ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+* OTHER DEALINGS IN THE SOFTWARE.
+*
+* Except as contained in this notice, the name of Maxim Integrated
+* Products, Inc. shall not be used except as stated in the Maxim Integrated
+* Products, Inc. Branding Policy.
+*
+* The mere transfer of this software does not imply any licenses
+* of trade secrets, proprietary technology, copyrights, patents,
+* trademarks, maskwork rights, or any other form of intellectual
+* property whatsoever. Maxim Integrated Products, Inc. retains all
+* ownership rights.
+*
+******************************************************************************/
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include "mxc_device.h"
+#include "led.h"
+#include "board.h"
+#include "mxc_delay.h"
+#include "uart.h"
+#include "rtc.h"
+#include "utils.h"
+
+#pragma GCC optimize("-O0")
+
+#define DEBUG_COMPORT MXC_UART0
+
+/***************************** VARIABLES *************************************/
+
+/************************    PUBLIC FUNCTIONS  *******************************/
+//void utils_delay_ms(uint32_t ms)
+//{
+//    MXC_Delay(ms * 1000UL);
+//}
```

Comment:
Is this function still in development or should this be deleted?

Reply:
Thanks, this is one leftover from the original CameraIF example that can be deleted.  Removing it now